plugins/outputs/health/health.go

```diff
@@ -153,6 +162,7 @@ func (h *Health) ServeHTTP(rw http.ResponseWriter, _ *http.Request) {
 
 // Write runs all checks over the metric batch and adjust health state.
 func (h *Health) Write(metrics []telegraf.Metric) error {
+	h.lastMetricTime = time.Now()
```

Comment:
Would be nice to add a comment below (line 174) stating why we don't need to check the metric timeout before calling `h.setHealthy()`...

Reply:
added